app/models/booking_response.rb

```diff
@@ -5,7 +5,7 @@ class BookingResponse
 
   attribute :visit
 
-  attribute :selection, Integer
+  attribute :selection, String
```

Comment:
Oh, that's really nice. I was looking at all my inquiry methods thinking that it ought to be possible to DRY them up.